source/slang/slang-ir-lower-generics.cpp

```diff
@@ -16,6 +16,94 @@
 
 namespace Slang
 {
+    void specializeRTTIObjectReferences(SharedGenericsLoweringContext* sharedContext)
```

Comment:
More comments would be more better. :)

I've been allowing a lot of this generics lowering and dynamic dispatch work to go in without extensive comments because it is important that we get it working, but I think it would be healthy for us to go back at some point and make sure it gets the kind of extensive comments that will make it easier (or even possible) to bring up new maintainers on this code.

I realize that we have a *lot* of code in the project that is impenetrable to anybody but the person who wrote it (e.g., a lot of the current type legalization stuff), but that is something I'd like us to push back on whenever we write new code.